### PR TITLE
Storage Manager docs updates

### PR DESCRIPTION
**Pull request by dbanksdesign**
#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are updated
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-storage/src/components/StorageManager/index.ts [hunk 1]
```diff
@@ -1,2 +1,9 @@
 export { StorageManager } from './StorageManager';
 export { StorageManagerProps } from './types';
+export {
+  DropZoneProps,
```
Comment (dbanksdesign) [inline packages/react-storage/src/components/StorageManager/index.ts:4]: Exporting these prop types to make the component overrides easier for customers

Comment (calebpollman) [reply packages/react-storage/src/components/StorageManager/index.ts:4]: Think you will need to add them to _src/components/index.ts_ and _src/index.ts_

Comment (dbanksdesign) [reply packages/react-storage/src/components/StorageManager/index.ts:4]: YOU will need to export them

Comment (calebpollman) [reply packages/react-storage/src/components/StorageManager/index.ts:4]: FINE

Comment (dbanksdesign) [reply packages/react-storage/src/components/StorageManager/index.ts:4]: done

**Review by calebpollman — APPROVED** [review]

**Pull request merged**